test/format/format_inline.h

```diff
@@ -320,7 +320,7 @@ static inline void
 wt_wrap_begin_transaction(WT_SESSION *session, const char *config)
 {
     WT_DECL_RET;
-    char transaction_config[MAX_FORMAT_PATH];
```

Comment:
I'd like to confirm if the buffer size is too short, that the call to `testutil_snprintf` will crash test format. I believe `snprintf` returns a number indicating how much size was used but we wrap that function in WT. So I don't know exactly what behaviour we'd get here.

What I want is that it does crash instead of creating an invalid buffer.

Reply:
I've double checked the code and we should actually be ok here. The underlying snprintf function will return an error if the buffer can't hold everything, and testutil_check will error out on that error. As such we won't be able to proceed with an invalid buffer.